<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rasviewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdi_area">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="export_menu">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="export_image_action"/>
     <addaction name="export_channel_action"/>
     <addaction name="export_document_action"/>
    </widget>
    <addaction name="open_action"/>
    <addaction name="close_action"/>
    <addaction name="export_menu"/>
    <addaction name="separator"/>
    <addaction name="print_action"/>
    <addaction name="separator"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="about_qt_action"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="zoom_mode_action"/>
    <addaction name="pan_mode_action"/>
    <addaction name="separator"/>
    <addaction name="zoom_in_action"/>
    <addaction name="zoom_out_action"/>
    <addaction name="reset_zoom_action"/>
    <addaction name="separator"/>
    <addaction name="home_axes_action"/>
    <addaction name="reset_axes_action"/>
    <addaction name="separator"/>
    <addaction name="status_bar_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="view_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_action"/>
   <addaction name="print_action"/>
   <addaction name="separator"/>
   <addaction name="zoom_mode_action"/>
   <addaction name="pan_mode_action"/>
   <addaction name="separator"/>
   <addaction name="zoom_in_action"/>
   <addaction name="zoom_out_action"/>
   <addaction name="reset_zoom_action"/>
   <addaction name="separator"/>
   <addaction name="home_axes_action"/>
   <addaction name="reset_axes_action"/>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a data file</string>
   </property>
   <property name="statusTip">
    <string>Open a data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="close_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close the currently selected data file</string>
   </property>
   <property name="statusTip">
    <string>Close the currently selected data file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="quit_action">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>&amp;About rasViewer</string>
   </property>
   <property name="toolTip">
    <string>Display information about the application</string>
   </property>
   <property name="statusTip">
    <string>Display information about the application</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="about_qt_action">
   <property name="text">
    <string>About QT</string>
   </property>
   <property name="toolTip">
    <string>Display information about the QT framework</string>
   </property>
   <property name="statusTip">
    <string>Display information about the QT framework</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="export_image_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Image...</string>
   </property>
  </action>
  <action name="export_channel_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Channel...</string>
   </property>
  </action>
  <action name="export_document_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Document...</string>
   </property>
  </action>
  <action name="reset_zoom_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="toolTip">
    <string>Show the entire dataset (reset all crop parameters to zero)</string>
   </property>
   <property name="statusTip">
    <string>Show the entire dataset (reset all crop parameters to zero)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="reset_axes_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset Axes</string>
   </property>
   <property name="toolTip">
    <string>Reset the axes scale and offset to their defaults</string>
   </property>
   <property name="statusTip">
    <string>Reset the axes scale and offset to their defaults</string>
   </property>
  </action>
  <action name="status_bar_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status &amp;Bar</string>
   </property>
  </action>
  <action name="zoom_in_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="toolTip">
    <string>Increase the zoom by 20%</string>
   </property>
   <property name="statusTip">
    <string>Increase the zoom by 20%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoom_out_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="toolTip">
    <string>Decrease the zoom by 20%</string>
   </property>
   <property name="statusTip">
    <string>Decrease the zoom by 20%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="print_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="toolTip">
    <string>Print the current image</string>
   </property>
   <property name="statusTip">
    <string>Print the current image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="zoom_mode_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Zoom Mode</string>
   </property>
   <property name="toolTip">
    <string>Click and drag on the image to select a zoom area</string>
   </property>
   <property name="statusTip">
    <string>Click and drag on the image to select a zoom area</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="pan_mode_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pan Mode</string>
   </property>
   <property name="toolTip">
    <string>Click and drag the image to move the zoomed area shown</string>
   </property>
   <property name="statusTip">
    <string>Click and drag the image to move the zoomed area shown</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="home_axes_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Home Axes</string>
   </property>
   <property name="toolTip">
    <string>Change the axis offset and scale so that the origin is at the upper left of the currently shown portion of the dataset</string>
   </property>
   <property name="statusTip">
    <string>Change the axis offset and scale so that the origin is at the upper left of the currently shown portion of the dataset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
